src/functions/template-tags/date.php

```diff
@@ -148,3 +148,172 @@ function tribe_get_datetime_separator( $default = ' @ ', $esc = false ) {
 		return apply_filters( 'tribe_datetime_separator', $separator );
 	}
 }//end if
+
+if ( ! function_exists( 'tribe_get_start_time' ) ) {
+	/**
+	 * Start Time
+	 *
+	 * Returns the event start time
+	 *
+	 * @category Events
+	 * @param int    $event       (optional)
+	 * @param string $dateFormat  Allows date and time formating using standard php syntax (http://php.net/manual/en/function.date.php)
+	 * @param string $timezone    Timezone in which to present the date/time (or default behaviour if not set)
+	 *
+	 * @return string|null Time
+	 */
+	function tribe_get_start_time( $event = null, $dateFormat = '', $timezone = null ) {
+		if ( is_null( $event ) ) {
+			global $post;
+			$event = $post;
+		}
+
+		if ( is_numeric( $event ) ) {
+			$event = get_post( $event );
+		}
+
+		if ( ! is_object( $event ) ) {
+			return;
+		}
+
+		if ( Tribe__Date_Utils::is_all_day( tribe_get_event_meta( $event->ID, '_EventAllDay', true ) ) ) {
+			return;
+		}
+
+		// @todo move timezones to Common
+		if ( class_exists( 'Tribe__Events__Timezones' ) ) {
+			$start_date = Tribe__Events__Timezones::event_start_timestamp( $event->ID, $timezone );
+		}
+
+		if ( '' == $dateFormat ) {
+			$dateFormat = tribe_get_time_format();
+		}
+
+		return tribe_format_date( $start_date, false, $dateFormat );
+	}
+}
+
+if ( ! function_exists( 'tribe_get_end_time' ) ) {
+	/**
+	 * End Time
+	 *
+	 * Returns the event end time
+	 *
+	 * @category Events
+	 * @param int    $event       (optional)
+	 * @param string $dateFormat  Allows date and time formating using standard php syntax (http://php.net/manual/en/function.date.php)
+	 * @param string $timezone    Timezone in which to present the date/time (or default behaviour if not set)
+	 *
+	 * @return string|null Time
+	 */
+	function tribe_get_end_time( $event = null, $dateFormat = '', $timezone = null ) {
+		if ( is_null( $event ) ) {
+			global $post;
+			$event = $post;
+		}
+
+		if ( is_numeric( $event ) ) {
+			$event = get_post( $event );
+		}
+
+		if ( ! is_object( $event ) ) {
+			return;
+		}
+
+		if ( Tribe__Date_Utils::is_all_day( tribe_get_event_meta( $event->ID, '_EventAllDay', true ) ) ) {
```

Comment:
At the minimum we'd need to 'guard' this like we're doing with calls to `Tribe__Events__Timezones` methods, still, gives me pause for thought on moving this into commons.